eml/components/login/RegisterForm.js

```diff
@@ -158,11 +163,7 @@ export default function RegisterForm() {
 
     try {
```

Comment:
try {
      await registerUser(obj)
        .then(async function (response) {
          // saves input information locally
          await saveUserInfoLocally(response._id, firstName, lastName, email);
          await AsyncStorage.setItem(USER_ID, response._id);
        }).then(async function () {
        .then(async function () {
          // logs in the user, if no errors occur, navigates to home screen and sets token
          await loginFromRegister(obj);
        }) 

keep this from staging!!! It was a hotfix from the user test